app/models/miq_action.rb

```diff
@@ -848,6 +848,31 @@ def action_container_image_analyze(action, rec, inputs)
     rec.scan
   end
 
+  def action_container_image_scan_all(action, rec, _inputs)
+    unless rec.kind_of?(ContainerImage)
+      MiqPolicy.logger.error("MIQ(#{__method__}): Unable to perform action [#{action.description}],"\
+                             " object [#{rec.inspect}] is not a Container Image")
+      return
+    end
+    ext = rec.ext_management_system
```

Comment:
Typical variable naming here is `ems` ... if you change it to `ems` it will be clearer across this method what this variable refers to.